Framework/PythonInterface/mantid/dataobjects/src/Exports/GroupingWorkspaceProperty.cpp

```diff
@@ -0,0 +1,18 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2024 ISIS Rutherford Appleton Laboratory UKRI,
+//   NScD Oak Ridge National Laboratory, European Spallation Source,
+//   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+// SPDX - License - Identifier: GPL - 3.0 +
+#include "MantidDataObjects/GroupingWorkspace.h"
+#include "MantidPythonInterface/api/WorkspacePropertyExporter.h"
+#include "MantidPythonInterface/core/GetPointer.h"
+
+using Mantid::DataObjects::GroupingWorkspace;
+
+GET_POINTER_SPECIALIZATION(Mantid::API::WorkspaceProperty<GroupingWorkspace>)
```

Comment:
Just out of curiosity, what does the `GET_POINTER_SPECIALIZATION` macro do?

Reply:
Looks like that macro is defined here with a comment, in case that helps: https://github.com/mantidproject/mantid/blob/main/Framework/PythonInterface/core/inc/MantidPythonInterface/core/GetPointer.h.